fabric/src/main/resources/fabric.mod.json

```diff
@@ -1,34 +1,67 @@
 {
-  "schemaVersion": 1,
-  "id": "now-playing",
-  "version": "${version}",
-  "name": "Now Playing",
-  "description": "Shows a 'now playing' popup whenever a song starts to play.",
-  "authors": [
-    "AppleTheGolden"
-  ],
-  "contact": {
-    "sources": "https://github.com/Scotsguy/now-playing"
-  },
-  "license": "MIT",
-  "icon": "assets/now_playing/icon.png",
-  "environment": "client",
-  "entrypoints": {
-    "client": [
-      "com.github.scotsguy.nowplaying.fabric.NowPlayingFabric"
+    "schemaVersion": 1,
+    "version": "${mod_version}",
+    "id": "${mod_id}",
+    "name": "${mod_name}",
+    "description": "${mod_description}",
+    "icon": "${mod_icon}",
+    "authors": [$mod_authors_list],
+    "contributors": [$mod_contributors_list],
+    "license": "${mod_license}",
+
+    "environment": "${mod_environment}",
+
+    "provides": [],
+
+    "entrypoints": {
+        "main": [],
+        "client": ["${mod_group}.${mod_id}.${mod_name}Fabric"],
+        "server": [],
+        "modmenu": ["${mod_group}.${mod_id}.gui.screen.ModMenuIntegration"]
+    },
+
+    "mixins": [
+        "${mod_id}.mixins.json",
+        "${mod_id}.fabric.mixins.json"
     ],
-    "modmenu": [
-      "com.github.scotsguy.nowplaying.fabric.ModMenuIntegration"
-    ]
-  },
-  "mixins": [
-    "now-playing.mixins.common.json",
-    "now-playing.mixins.fabric.json"
-  ],
-  "depends": {
-    "fabricloader": ">=0.9.0",
-    "minecraft": "1.20.x",
-    "fabric": "*",
-    "cloth-config2": ">=11.0"
-  }
+
+    "accessWidener": "${mod_id}.accesswidener",
+
+    "depends": {
+        "java": [$java_versions_fabric_list],
+        "minecraft": [$minecraft_versions_fabric_list],
+        "fabricloader": [$fabric_loader_versions_list],
+        "fabric-lifecycle-events-v1": [$fabric_api_versions_list]
+    },
```

Comment:
I'm 90% sure that resource loader is also required, otherwise the popups don't actually tell you the title of the song.

Reply:
Sounds right, that being the case would you prefer a direct reference to `fabric-api`?

Reply:
That's how it was before, and it's probably fine. No one downloads parts of fabric api individually, really.

Reply:
Sounds good, it allows taking advantage of other mods that JiJ it, but that's a minor thing.